File: codes/java/src/inheritance/single/Student.java
package inheritance.single;

public class Student extends Person {

    private int grade;

    public Student(String name, int grade) {
        super(name);
        this.grade = grade;
    }

    public void info(){
        super.info();
        System.out.println("My grade is: " + grade);
    }

}

File: codes/python/inheritance/single/student.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-

from person import Person

class Student(Person):

    def __init__(self, name, grade):
        Person.__init__(self, name)
        #super(Student, self).__init__(name)#the same
        self.grade = grade

    def info(self):
        super(Student, self).info()
        #Person.info(self) #the same
        print("My grade: " + str(self.grade))
